include/dba.php

```diff
@@ -1235,16 +1225,32 @@ public static function count($table, array $condition = [])
 	 * @param array $condition
 	 * @return string
 	 */
-	private static function buildCondition(array &$condition = [])
+	private static function buildCondition(&$condition = [])
```

Comment:
What is the case when a non-array would be passed to this function?

Reply:
Then nothing is passed - for example a "select" without a condition.

Reply:
Then there shouldn't be a call to `buildCondition` but the type hint should stay and the type check should be removed.

Reply:
I just checked the two places `buildCondition` is called and in both places the default value is an array.